tests/Responses/Expected/product_attribute/create_select_response.json

```diff
@@ -3,11 +3,7 @@
     "code": "mug_color",
     "type": "select",
     "configuration": {
-        "choices": {
-            "yellow": "yellow",
-            "green": "green",
-            "black": "black"
-        },
+        "choices": @array@,
```

Comment:
IMO we should change it to something like:
```
{
 "@string@": "yellow",
 "@string@": "green",
 // etc..
}
```

Reply:
I know and I agree that current example doesn't check behaviour properly, but this feature is not implemented in php-matcher (there is even an issue: https://github.com/coduo/php-matcher/issues/92)

Reply:
What about adding an additional assertion in the test file, eg. asserting that the response matches this file contents +  getting `$response['configuration']['choices']` and asserting they are what we expect (like https://github.com/Sylius/SyliusElasticSearchPlugin/blob/master/tests/Controller/SearchControllerApiTest.php#L436)?

Reply:
Done 👍